.gitignore

```diff
@@ -0,0 +1 @@
+.DS_Store
```

Comment:
It's possible to apply this globally on a Mac so you don't need to add it to each repo 🎉 

```bash
cat ~/.gitexcludes
.DS_Store
```

Reply:
good tip, thanks